matbench_discovery/metrics.py

```diff
@@ -83,20 +98,17 @@ def stable_metrics(
             Recall, Accuracy, F1, TPR, FPR, TNR, FNR, MAE, RMSE, R2.
     """
     n_true_pos, n_false_neg, n_false_pos, n_true_neg = map(
-        sum, classify_stable(each_true, each_pred, stability_threshold)
+        sum, classify_stable(each_true, each_pred, stability_threshold, fillna)
     )
 
     n_total_pos = n_true_pos + n_false_neg
     n_total_neg = n_true_neg + n_false_pos
     # prevalence: dummy discovery rate of stable crystals by selecting randomly from
     # all materials
-    prevalence = n_total_pos / len(each_true)
+    prevalence = n_total_pos / (n_total_pos + n_total_neg)
```

Comment:
you may wish to revert this I'm not 100% sure whether the prevalence should adjust